CHANGES.md

```diff
@@ -0,0 +1,16027 @@
+   project.
+   (CVE-2018-0739)
+
+   *Matt Caswell*
+
+ * Incorrect CRYPTO_memcmp on HP-UX PA-RISC
+
+   Because of an implementation bug the PA-RISC CRYPTO_memcmp function is
+   effectively reduced to only comparing the least significant bit of each
+   byte. This allows an attacker to forge messages that would be considered as
+   authenticated in an amount of tries lower than that guaranteed by the
+   security claims of the scheme. The module can only be compiled by the
+   HP-UX assembler, so that only HP-UX PA-RISC targets are affected.
+
+   This issue was reported to OpenSSL on 2nd March 2018 by Peter Waltenberg
+   (IBM).
+   (CVE-2018-0733)
+
+   *Andy Polyakov*
+
+ * Add a build target 'build_all_generated', to build all generated files
+   and only that.  This can be used to prepare everything that requires
+   things like perl for a system that lacks perl and then move everything
+   to that system and do the rest of the build there.
+
+   *Richard Levitte*
+
+ * Backport SSL_OP_NO_RENGOTIATION
+
+   OpenSSL 1.0.2 and below had the ability to disable renegotiation using the
+   (undocumented) SSL3_FLAGS_NO_RENEGOTIATE_CIPHERS flag. Due to the opacity
+   changes this is no longer possible in 1.1.0. Therefore the new
+   SSL_OP_NO_RENEGOTIATION option from 1.1.1-dev has been backported to
+   1.1.0 to provide equivalent functionality.
+
+   Note that if an application built against 1.1.0h headers (or above) is run
+   using an older version of 1.1.0 (prior to 1.1.0h) then the option will be
+   accepted but nothing will happen, i.e. renegotiation will not be prevented.
+
+   *Matt Caswell*
+
+ * Removed the OS390-Unix config target.  It relied on a script that doesn't
+   exist.
+
+   *Rich Salz*
+
+ * rsaz_1024_mul_avx2 overflow bug on x86_64
+
+   There is an overflow bug in the AVX2 Montgomery multiplication procedure
+   used in exponentiation with 1024-bit moduli. No EC algorithms are affected.
+   Analysis suggests that attacks against RSA and DSA as a result of this
+   defect would be very difficult to perform and are not believed likely.
+   Attacks against DH1024 are considered just feasible, because most of the
+   work necessary to deduce information about a private key may be performed
+   offline. The amount of resources required for such an attack would be
+   significant. However, for an attack on TLS to be meaningful, the server
+   would have to share the DH1024 private key among multiple clients, which is
+   no longer an option since CVE-2016-0701.
+
+   This only affects processors that support the AVX2 but not ADX extensions
+   like Intel Haswell (4th generation).
+
+   This issue was reported to OpenSSL by David Benjamin (Google). The issue
+   was originally found via the OSS-Fuzz project.
+   (CVE-2017-3738)
+
+   *Andy Polyakov*
+
+### Changes between 1.1.0f and 1.1.0g [2 Nov 2017] ###
+
+ * bn_sqrx8x_internal carry bug on x86_64
+
+   There is a carry propagating bug in the x86_64 Montgome
```

Comment:
Use backquotes around `__owur`.  This starts a looooooooooooooooooooong bold chunk of text

Reply:
Hmmm... there is only a single __owur in the text and that looks perfectly normal. Are you using a different markdown viewer?

![image](https://user-images.githubusercontent.com/11467854/70086442-b4fe2b80-1612-11ea-8cc8-3970835c8397.png)

Reply:
Note: the CHANGES file contains a lot more text than the NEWS file, so I did not manage yet to go through the entire text and spot all formatting issues.

Reply:
@levitte are you still seeing this bold chunk of text in the current version of the [CHANGES](https://github.com/mspncp/openssl/blob/revamp-the-docs/CHANGES.md) file?